.github/workflows/qiita-ci.yml

```diff
@@ -118,12 +120,17 @@ jobs:
           COVER_PACKAGE: ${{ matrix.cover_package }}
         run: |
           conda activate qiita_client
-          export QIITA_SERVER_CERT=`pwd`/qiita-dev/qiita_core/support_files/server.crt
+          export QIITA_SERVER_CERT=`pwd`/qiita-dev/qiita_core/support_files/ci_server.crt
+          export QIITA_ROOT_CA=`pwd`/qiita-dev/qiita_core/support_files/ci_rootca.crt
           export QIITA_CONFIG_FP=`pwd`/qiita-dev/qiita_core/support_files/config_test_local.cfg
-
-          nosetests --with-doctest --with-coverage --cover-package=qiita_client
-
-      - uses: codecov/codecov-action@v1
+          export PYTHONWARNINGS="ignore:Certificate for localhost has no \`subjectAltName\`"
+
+          # before starting nosetests, add the root ca used to sign qiita's ci_server.crt file
+          # to the environment's certifi store. This will prevent CERTIFICATE_VERIFY_FAILED
+          # errors.
+          python rootca_insert.py $QIITA_ROOT_CA
+          nosetests --with-doctest --with-coverage -v --cover-package=qiita_client 
```

Comment:
Is this necessary? I don't remember having to do this in any of the current plugins?

Reply:
Removing both as we are removing rootca_insert.py and we don't need to run nose tests for qiita.